packages/boiler-task-selenium/package.json

```diff
@@ -35,6 +35,7 @@
     "mocha": "^2.4.5",
     "require-hacker": "^2.1.3",
     "selenium-standalone": "^4.9.1",
-    "webdriverio": "^3.3.0"
+    "wdio-mocha-framework": "^0.2.12",
+    "webdriverio": "^4.0.5"
   }
```

Comment:
@stephenkao you don't have to physically bump the `package.json` `lerna` will bump it automatically